src/modules/comment/controller.js

```diff
@@ -0,0 +1,158 @@
+const sanitizeHtml = require('sanitize-html')
+const Comment = require('../../models/comment')
+
+let _this
```

Comment:
This is a old OOP pattern used mostly as workarounds for specific reasons. Is there any reason we need to use it here?